mautic/mautic
# Update country name "Turkey" to "Türkiye"

| Q                                      | A
| -------------------------------------- | ---
| Bug fix? (use the a.b branch)          | 🟢?
| New feature/enhancement? (use the a.x branch)      | 🔴
| Deprecations?                          | 🔴
| BC breaks? (use the c.x branch)        | 🔴
| Automated tests included?              | 🔴
| Related user documentation PR URL      | /
| Related developer documentation PR URL | /
| Issue(s) addressed                     | Fixes #13996 


## Description

The country formerly know as "Turkey" is now officially called "Türkiye". This PR implements that change into Mautic.

As part of that, the function that gets the correct flag image for each country can now handle umlauts. This will come in handy for countries like the "Åland Islands".


---
### 📋 Steps to test this PR:

1. Open this PR on Gitpod or pull down for testing locally (see docs on testing PRs [here](https://contribute.mautic.org/contributing-to-mautic/tester))
2. Create a new contact
3. See that the country "Türkiye" is visible and selectable.
![image](https://github.com/user-attachments/assets/850594c7-744c-41ed-a210-19b950ee9f45)

5. Add a region for the country "Türkiye".
![image](https://github.com/user-attachments/assets/07553a34-5ebd-46fb-a8de-cb51c8eb2a1a)

7. Save the contact and verify that the Turkish flag is shown in the overview
![image](https://github.com/user-attachments/assets/55a59bea-e9f7-4816-8e17-6efc13d778c9)

8. Ideally, also verify that the new migration changes "Turkey" to "Türkiye" in the "country" column in the "leads" table.
